source/3d-secure-2.html.erb

```diff
@@ -0,0 +1,132 @@
+      <p class="govuk-body">Services are still liable for transactions that are exempt from 3DS2 and are responsible if a user disputes a transaction on the basis that they did not receive the goods or service they paid for.</p>
+
+      <h2 class="govuk-heading-m">How to set up 3D Secure 2</h2>
+      <p class="govuk-body">The way you set up 3DS2 will depend on your Payment Service Provider (PSP).</p>
+
+      <h3 class="govuk-heading-s">Stripe</h3>
+      <p class="govuk-body">GOV.UK Pay is managing the 3DS2 integration with Stripe so you do not need to make any changes to your integration with us. Stripe will automatically apply exemptions.</p>
+
+      <h3 class="govuk-heading-s">Worldpay</h3>
+      <p class="govuk-body">Worldpay are implementing 3DS2 as part of their Flex product.. You will receive 3DS Flex credentials from Worldpay and <a class="govuk-link" href="https://docs.payments.service.gov.uk/set_up_3dsecure/#set-up-3ds2-3ds-flex">you will need to add them into your Pay service.</a></p>
+      <p class="govuk-body">If you are a Government Banking customer, Government Banking will contact you to let you know when to enable 3DS Flex on your account.</p>
+      <p class="govuk-body">If you have a direct contract with Worldpay (not via Government Banking), Worldpay will send you your 3DS credentials.</p>
+      <p class="govuk-body">Worldpay will send all transactions via 3DS2 or 3DS. Exemptions (see above) will not be applied. If you’d like to apply for exemptions for your transactions, <a class="govuk-link" href="https://www.payments.service.gov.uk/support/">contact GOV.UK Pay.</a></p>
+
+      <h3 class="govuk-heading-s">Barclays ePDQ</h3>
+      <p class="govuk-body">GOV.UK Pay has built the integration with 3DS2 for ePDQ. You will not need to make any changes to your integration. Exemptions (see above) will not be applied.</p>
+
+      <h3 class="govuk-heading-s">Barclays Smartpay</h3>
+      <p class="govuk-body">Barclaycard plan to decommission their Bureau ‘Smartpay A’ payment gateway in 2021. Given those timescales we have not built support for 3DS2. You can enable 3DS which meets the requirements of SCA.</p>
+
+      <h2 class="govuk-heading-m">Costs for 3D Secure 2</h2>
+      <p class="govuk-body">GOV.UK Pay will not pass on any of the costs of implementing 3D Secure 2. However, payment charges will vary depending on your PSP.</p>
+      <p class="govuk-body">There’s no extra cost if you’re with Stripe.</p>
+      <p class="govuk-body">Check with Worldpay, Barclays Smartpay and Barclays ePDQ for details of their individual costs. If you are a Government Banking customer, contact serviceteam.gbs@hmrc.gov.uk.</p>
+
+      <h2 class="govuk-heading-m">Using digital wallets</h2>
+      <p class="govuk-body">There is no extra cost for transactions using digital wallets because they meet the requirements for 3DS2 authentication. You may want to make sure this option is switched on in your settings.</p>
```

Comment:
"You may want to make sure this option is switched on in your settings." => "Check that you have enabled this option in your settings."